### ISSUE TITLE
False positive for HLQ002

### ISSUE THREAD

**Issue by robinvanpoppel** [NetFabric/NetFabric.Hyperlinq.Analyzer#34]
Hi!

The following code reports a false positive for rule `HLQ002`. I guess it has to do with the return type. Being a string, it does implement `IEnumerable<char>`, but returning a null in these kinds of methods should normally be fine in my opinion.

```csharp
namespace ReproHLQ002
{
    public static class ClaimsExtensions
    {
        public static string SomeMethod()
        {
            const string x = "";
            if (string.IsNullOrEmpty(x))
            {
                // analyzer reports HLQ002 (Enumerable cannot be null. Return an empty enumerable instead.) on the next line
                return null;
            }

            return x;
        }
    }
}
```

Thanks for your time!

**Comment by aalmada** [comment]
Hi @robinvanpoppel 
You are right. What version are you using? I thought I had fixed this issue with #29

**Comment by robinvanpoppel** [comment]
@aalmada, my bad. I copied the PackageReference bit from the readme, which uses the 1.* wildcard, so I was under the impression that I was on the latest version already.

**Comment by aalmada** [comment]
Oh! I forgot about the README. Thanks for the PR.
Feel free to suggest improvements, report more issues, and submit PRs.
